crates/factor-outbound-pg/src/client.rs

```diff
@@ -0,0 +1,75 @@
+use native_tls::TlsConnector;
+use postgres_native_tls::MakeTlsConnector;
+use spin_world::async_trait;
+use tokio_postgres::{config::SslMode, types::ToSql, Error, Row, ToStatement};
+use tokio_postgres::{Client as PgClient, NoTls, Socket};
```

Comment:
Nit

```suggestion
use tokio_postgres::{Client as TokioClient, NoTls, Socket};
```

I prefer `TokioClient` as a name since presumably all clients are pg clients, no?